modules/tinymce/src/themes/silver/main/ts/ui/window/SilverInlineDialog.ts

```diff
@@ -24,11 +24,23 @@ interface RenderedDialog<T extends Dialog.DialogData> {
   readonly instanceApi: Dialog.DialogInstanceApi<T>;
 }
 
+const getInlineDialogSizeClasses = (size: Dialog.DialogSize): string[] => {
```

Comment:
```suggestion
const getInlineDialogSizeClasses = (size: Dialog.DialogSize): Optional<string> => {
```
Does this need to be an array if there's only one possible dialog size at a time? You could make this an optional and return `Optional.none()` in the default case